.readthedocs.yaml

```diff
@@ -1,13 +1,30 @@
-version: "2"
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
 build:
-  os: "ubuntu-24.04"
+  os: ubuntu-22.04
   tools:
-    python: "3.12"
-
-python:
-  install:
-    - requirements: docs/requirements.txt
+    python: "3.11"
```

Comment:
Is there a reason 3.11 is needed here? The rest of Jubilant requires 3.12, so I was thinking this should match.